plugins/poll/app/models/poll.rb

```diff
@@ -47,6 +67,36 @@ def can_see_voters?(user)
   def ranked_choice?
     type == "ranked_choice"
   end
+
+  def self.preload!(polls, user_id: nil)
```

Comment:
I think we should accept only a scope here and use that scope in the queries below.